distributed/protocol/rmm.py

```diff
@@ -0,0 +1,22 @@
+import rmm
+from .cuda import cuda_serialize, cuda_deserialize
+
+
+@cuda_serialize.register(rmm.DeviceBuffer)
+def serialize_rmm_device_buffer(x):
+    header = x.__cuda_array_interface__.copy()
+    frames = [x]
+    return header, frames
+
+
+@cuda_deserialize.register(rmm.DeviceBuffer)
+def deserialize_rmm_device_buffer(header, frames):
+    (arr,) = frames
+
+    # Copy data into new `DeviceBuffer` if needed
+    if not isinstance(arr, rmm.DeviceBuffer):
+        (ptr, _) = arr.__cuda_array_interface__["data"]
+        (size,) = header["shape"]
+        arr = rmm.DeviceBuffer(ptr=ptr, size=size)
```

Comment:
Not actually sure this special case is necessary/possible. If RMM is available, we already [allocate a `DeviceBuffer`]( https://github.com/dask/distributed/blob/4a8a4f3bce378406e83a099e8a12fc9bc12ef25c/distributed/comm/ucx.py#L50 ) and [fill it]( https://github.com/dask/distributed/blob/4a8a4f3bce378406e83a099e8a12fc9bc12ef25c/distributed/comm/ucx.py#L188 ). So it might be ok to drop this code. Left it in as a preventive measure just in case things get changed around later.